### ISSUE TITLE
Create a labelled test set to evaluate layout analysis methods

### ISSUE THREAD

**Issue by kdutia** [climatepolicyradar/navigator#34]
**find and deploy tool**
- [x] Do a review of the tools that exist and identify the best one
- [x] Deploy to AWS 

**create labelled dataset** 
- [x] Create unlabelled image set in s3
- [x] Define labels to be used based on alpha scope (and cross-check with datasets)
- [x] Set these labels up in label-studio
- [ ] Create labelled test set

**Comment by kdutia** [comment]
https://github.com/heartexlabs/awesome-data-labeling as well as other resources contain lists of data labelling tools. I recommend we use [label-studio](https://github.com/heartexlabs/label-studio#integrate-label-studio-with-your-existing-tools), as it's flexible, open-source, and multimodal. We could use this tool in future for any NLP or OCR labelling we wanted to do. It also supports active learning.

**Comment by kdutia** [comment]
label-studio deployed to an EC2 machine on AWS. Currently set to the same on/off schedule as the prototype, by giving it the tag `deployment: prototype`.

**Comment by kdutia** [comment]
Labelled a few examples in label-studio and compared these ground truth labels against labels predicted by [a mask-RCNN model trained on PubLayNet from the Layout Parser library](https://layout-parser.readthedocs.io/en/latest/notes/modelzoo.html).

There were no easily-accessible pretrained models on DocBank, which was the other dataset I wanted to compare predictions against. Found code for chargrid-based models also doesn't come with pretrained models.

[Labels we've defined](https://www.notion.so/climatepolicyradar/Document-Layout-Analysis-R-D-ed3de49d21bc440691bcd56937d56978#913a8386b2c44c6383601bed8829c060) look sensible to measure the performance of PubLayNet trained models.

Next steps are to create a test set for English language PDFs, stratified across document type and geography, and to begin labelling this test set.

**Comment by kdutia** [comment]
Created a labelled test set of 60 pages on label-studio, with the following characteristics:

* classes labelled were heading; text; figure; table; list; caption; header text; footer text; footnote; page number
* 42 from executive docs; 18 from legislative docs
* 57 portrait; 3 landscape

This test set was used to complete the work in #36. A new one should be created following the guidance in that issue.
